src/mono/mono/metadata/custom-attrs.c

```diff
@@ -2488,8 +2488,8 @@ mono_reflection_get_custom_attrs_info_checked (MonoObjectHandle obj, MonoError *
 		cinfo = mono_custom_attrs_from_builders_handle (NULL, &dynamic_image->image, cattrs);
 	} else if (strcmp ("MonoGenericClass", klass_name) == 0) {
 		MonoReflectionGenericClassHandle gclass = MONO_HANDLE_CAST (MonoReflectionGenericClass, obj);
-		MonoReflectionTypeHandle generic_type = MONO_HANDLE_NEW_GET (MonoReflectionType, gclass, generic_type);
-		cinfo = mono_reflection_get_custom_attrs_info_checked (MONO_HANDLE_CAST (MonoObject, generic_type), error);
+		MonoReflectionTypeHandle _genericType = MONO_HANDLE_NEW_GET (MonoReflectionType, gclass, _genericType);
```

Comment:
I think `generic_type` aligns more with the style of the mono codebase, but I defer to a mono dev for the official word.

Reply:
Agree, I can update it if mono folks prefer to keep this naming.

Reply:
@buyaa-n yea, let's put it back.

Reply:
Done